src/Middleware/OutputCaching/src/OutputCacheEntry.cs

```diff
@@ -69,26 +66,9 @@ public bool TryFindHeader(string key, out StringValues values)
     internal void SetBody(ReadOnlySequence<byte> value, bool recycleBuffers)
     {
         Body = value;
-        _recycleBuffers = recycleBuffers;
-    }
-
-    public void Dispose()
-    {
-        var headers = Headers;
-        var body = Body;
-        Headers = default;
-        Body = default;
-        Recycle(headers);
```

Comment:
This is also memory coming from the ArrayPool that we're no longer returning?

Reply:
resolved https://github.com/dotnet/aspnetcore/pull/51067/commits/0275ad28f4b4b8e5985c71d9f61a889f9c5386ce